samples/51.teams-messaging-extensions-action/src/main/java/com/microsoft/bot/sample/teamsaction/TeamsMessagingExtensionsActionBot.java

```diff
@@ -78,7 +78,6 @@
         card.setText(action.getMessagePayload().getBody().getContent());
 
         CardImage cardImage = new CardImage();
```

Comment:
This should be inside the `if` sentence too

Reply:
Ack, but this is used in both if sentences so I will have to declare it two times